tests/service/connectors/test_queryconfig.py

```diff
@@ -70,45 +70,57 @@ def found_query_keys(node: TraversalNode, values: Dict[str, Any]) -> Set[str]:
         }
 
         # values exist for all query keys
-        assert found_query_keys(
```

Comment:
No changes here but I got tired of having to omit this auto-format from my commits.